daal4py/sklearn/utils/readcsv.py

```diff
@@ -0,0 +1,121 @@
+from pathlib import Path
```

Comment:
i guess this would be considered publicAPI for daal4py. 
I don't think that this is functionality that we would like  to expose?

Reply:
I don't have a good solution for not putting it into a package _and_ making it reusable. This is a limitation in Python's import system for which I don't have a good workaround. Maybe @samir-nasibli has an idea.

Initially that was why I created the `tests` namespace inside `daal4py` - I didn't think it would hurt there.

Reply:
but you can put it just in to tests folder in the root of repo? 
or we have https://github.com/intel/scikit-learn-intelex/tree/master/scripts already 

@Alexsandruss - might be any suggestions?

Reply:
Looks good overall